client/post-editor/editor-sidebar/feedback-list.jsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, PureComponent } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Accordion from 'components/accordion';
+import Gravatar from 'components/gravatar';
+
+class FeedbackList extends PureComponent {
```

Comment:
we've had a good number of discussions on this and I don't know where we ended up with `Component` vs `PureComponent`. `react-redux` already makes a component pure by default so I tend to discourage mixing both pure bases (using `Component` instead) but since we're not wrapped with the Redux helper I suppose this here is appropriate.